benchmarks/libpng-1.2.56/build.sh

```diff
@@ -14,11 +14,28 @@
 # limitations under the License.
 
 tar xf libpng-1.2.56.tar.gz
+tar xf zlib-1.2.11.tar.gz
 
-cd libpng-1.2.56
+OLDCXXFLAGS=${CXXFLAGS}
```

Comment:
Summarizing what we discussed over VC here:
The only way a builder image can affect a runner image is by putting htings in $OUT. 
Since this doesn't put the zlib build in $OUT is essentially a noop.
Maybe the zlib install is only needed for building. Or maybe it is already installed in the runner image so at runtime, the system zlib is used.

Reply:
Summarizing final conclusion:

We're taking this out for now and will come up with a more generic solution. We'll work on integrating libpng with klee in a separate PR.